src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -1197,6 +1202,14 @@ private IEnumerable<TypeInferenceInfo> InferTypeInBaseMethodDeclaration(BaseMeth
                     : SpecializedCollections.EmptyEnumerable<TypeInferenceInfo>();
             }
 
+            private IEnumerable<TypeInferenceInfo> InferTypeInGetAccessorDeclaration(AccessorDeclarationSyntax accessorDeclaration)
```

Comment:
this is same as above, consider removing duplication

Reply:
I extracted the duplication to a separate method.